public/estadoDeRegistro.html

```diff
@@ -49,24 +49,16 @@
       </div>
     </header>
     <main>
-      <div class="container">
-        <div class="subtitles">
-          <br /><br />
-          <h2>Solicitudes de regisro de activos</h2>
-          <br />
-          <div class="buscarActivos">
-            <input
-              type="search"
-              placeholder="Nombre o ID de activo"
-              id="buscarNombreActivo"
-            />
-          </div>
-          <div class="fecha">
-            <input type="date" id="buscarFecha" />
-          </div>
-          <div class="botonBuscar">
-            <button>Buscar</button>
-          </div>
+      <br /><br />
+      <h2 class="subtitles">Solicitudes de regisro de activos</h2>
+      <br />
+      <div>
+        <div class="inputs">
+          <label for="text">Nombre o ID de activo</label>
```

Comment:
```suggestion
          <label for="txtBuscarActivo">Nombre o ID de activo</label>
```
El `for` siempre hace referencia al id del input al que está asociado

Reply:
Listo